MpsAllocatorSample/Program.cs

```diff
@@ -114,17 +114,21 @@ static async Task ListBuildSummaries()
             }
             else
             {
-                PrettyPrintJson(res.Result); 
+                PrettyPrintJson(res.Result);
             }
         }
-        
+
         static async Task ListMultiplayerServers()
         {
             var req = new PlayFab.MultiplayerModels.ListMultiplayerServersRequest();
             string buildID = ReadBuildIDFromInput();
             var regions = await GetRegions(buildID);
-            Console.WriteLine($"Enter region (options are {string.Join(",", regions)})");
-            string region = Console.ReadLine();
+            string region = regions.FirstOrDefault();
+            if (regions.Count() > 1)
```

Comment:
are we auto-picking a region here? If so, maybe we can tell the user (Console.WriteLine) that we are using a specific region

Reply:
also, can we have that change specified in the PR description so it's better tracked?

Reply:
Yes, and I think I did that in a later commit.